app/models/validations/sales/setup_validations.rb

```diff
@@ -2,14 +2,22 @@ module Validations::Sales::SetupValidations
   include Validations::SharedValidations
   include CollectionTimeHelper
 
-  def validate_saledate(record)
+  def validate_saledate_collection_year(record)
     return unless record.saledate && date_valid?("saledate", record) && FeatureToggle.saledate_collection_window_validation_enabled?
 
     unless record.saledate.between?(active_collection_start_date, current_collection_end_date)
       record.errors.add :saledate, saledate_validation_error_message
     end
   end
 
+  def validate_saledate_two_weeks(record)
+    return unless record.saledate && date_valid?("saledate", record) && FeatureToggle.saledate_two_week_validation_enabled?
+
+    if record.saledate > Time.zone.today + 14
```

Comment:
```suggestion
    if record.saledate > Time.zone.today + 14.days
```

Same, but a little clearer :)

Reply:
yep agree, I'll copy this back to lettings too